Missionframework/stringtable.xml

```diff
@@ -345,6 +345,10 @@
             <Spanish>- DESCARGA CAJA DE COMBUSTIBLE -</Spanish>
             <Turkish>- BENZIN KUTUSUNU ÇIKAR -</Turkish>
         </Key>
+        <Key ID="STR_KPLIB_ACTION_PERMISSION">
+             <English>-- Permissions --</English>
```

Comment:
```suggestion
            <English>-- Permissions --</English>
```

Reply:
It's indeed the `suggestions beta` from GitHub. If you hit the suggestion button in the comment field, it adds one more space, which has to be removed. So let's realign this ^^